Mathlib/Data/Finset/Basic.lean

```diff
@@ -1916,7 +1916,7 @@ theorem union_sdiff_self (s t : Finset α) : (s ∪ t) \ t = s \ t :=
 
 -- TODO: Do we want to delete this lemma and `Finset.disjUnion_singleton`,
 -- or instead add `Finset.union_singleton`/`Finset.singleton_union`?
-theorem sdiff_singleton_eq_erase (a : α) (s : Finset α) : s \ singleton a = erase s a := by
+theorem sdiff_singleton_eq_erase (a : α) (s : Finset α) : s \ {a} = erase s a := by
```

Comment:
If you want us to be more opinionated about this pattern, should this be `@[simp]`?

Reply:
I think the library is already pretty in favour of this pattern, and this PR is more about conforming to it rather than creating it. That said, I'm hesitant to make this simp, because it's often useful in the reverse direction when doing proofs (see eg `erase_eq`, which is the opposite rewrite).